private/model/api/operation.go

```diff
@@ -253,8 +258,11 @@ func (o *Operation) GoCode() string {
 
 	if len(o.OutputRef.Shape.EventStreamsMemberName) != 0 {
 		// TODO need better was of updating protocol unmarshalers
+		o.API.imports["fmt"] = true
```

Comment:
Can go away with removal of `fmt.Errorf`

Reply:
Moved to eventstream template generation, needed for a few error messages there.